lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -0,0 +1,251 @@
+                if dirent.name.startswith("tools-") or dirent.name == "tmp":
+                    continue
+
+                no_of_tb += 1
+                result_dir = dirent
+
+                try:
+                    mrt = MakeResultTb(
+                        result_dir,
+                        temp_dir,
+                        self.context.controller,
+                        self.config,
+                        self.logger,
+                    )
+                except MakeResultTb.AlreadyCopied:
+                    self.logger.info(f"Already copied {result_dir}")
+                    continue
+                except MakeResultTb.BenchmarkRunning:
+                    self.logger.warning(
+                        f"Skipping {result_dir}: the benchmark appears to be"
+                        " running.  If that's incorrect, remove the"
+                        f" {result_dir}/.running directory and try again"
+                    )
+                    continue
+                except (NotADirectoryError, FileNotFoundError) as exc:
+                    self.logger.error(str(exc))
+                    failures += 1
+                    continue
+
+                try:
+                    result_tb_name, result_tb_len, result_tb_md5 = mrt.make_result_tb(
+                        single_threaded=single_threaded
+                    )
+                except BadMDLogFormat as exc:
+                    self.logger.warning(str(exc))
+                    failures += 1
+                    continue
+                except FileNotFoundError as exc:
+                    self.logger.error(str(exc))
+                    failures += 1
+                    continue
+                except RuntimeError as exc:
+                    self.logger.warning("Error encountered making tar ball, '%s'", exc)
+                    failures += 1
+                    continue
+                except Exception as exc:
+                    self.logger.error(
+                        "Unexpected error occurred making tar ball for '%s', '%s'",
+                        result_dir,
+                        exc,
+                    )
+                    failures += 1
+                    continue
+
+                try:
+                    crt = CopyResultTb(
+                        self.context.controller,
+                        result_tb_name,
+                        result_tb_len,
+                        result_tb_md5,
+                        self.config,
+                        self.logger,
+                    )
+                    crt.copy_result_tb(self.context.token)
+                except Exception as exc:
+                    if isinstance(exc, (CopyResultTb.FileUploadError, RuntimeError)):
+                        msg = "Error uploading file"
+                    else:
+                        msg = "Unexpected error occurred copying tar ball remotely"
+                    self.logger.error(f"{msg}, '%s', %s", result_tb_name, exc)
```

Comment:
Wow, that sure does look weird, combining the modern f-string syntax with the old-fashioned printf default `Logger` formatting. I'm tempted to suggest that, in this case, it would be better to defer formatting `msg` to a `%s` as part of the `Logger` formatting.

Reply:
Yeah, that or do it all up-front with the f-string...but, I concur that mixing them seems less than desirable.

Take yur pick!
```
                    self.logger.error(f"{msg}, '%s', %s", result_tb_name, exc)
                    self.logger.error("%s, '%s', %s", msg, result_tb_name, exc)
                    self.logger.error("{}, '{}', {}", msg, result_tb_name, exc)
                    self.logger.error(f"{msg}, {result_tb_name!r}, {exc!s}")
```

Reply:
We don't have that option, unfortunately.

We are in a middle ground -- while the server side has a logger that supports the use of "braces" formatting, the agent side does not have that code.  We really need to convert the agent side to share the same logger.

However, that is for another PR.

Alas, for the agent side, we can only use "%" formatting.

Hence, the layout above.

Reply:
Um, OK, so the third of the four options that I offered is not viable...but why wouldn't the others (which are, I think, better than the one that you've proffered) work?